wordpress-mobile/WordPress-Android
# Feature/enable seen unseen feature

This PR enables an unread post count feature and seen/unseen toggle.

To test:
- Install the app using vanilla flavor.
- Make sure you haven't manually overridden the flag.
- Confirm that the unread post count is visible in Reader's Following site filter.
- Confirm that the seen/unseen button is visible where available.

PR submission checklist:
